src/services/nodeApp.ts

```diff
@@ -0,0 +1,14 @@
+import * as admin from 'firebase-admin'
+
+if (!admin.apps.length) {
+  admin.initializeApp({
```

Comment:
@DimitarTsvetkov this function should allow us to access firebase admin functionalities.